data/items/items.xml

```diff
@@ -30115,43 +30148,190 @@
+	</item>
+	<item id="23480" name="the remains of an elemental">
+		<attribute key="containerSize" value="11" />
+		<attribute key="weight" value="10000" />
+		<attribute key="decayTo" value="0" />
+		<attribute key="duration" value="500" />
+		<attribute key="corpseType" value="venom" />
+		<attribute key="fluidSource" value="slime" />
+	</item>
+	<item id="23481" name="the remains of an elemental">
+		<attribute key="weight" value="25000" />
+		<attribute key="containerSize" value="11" />
+		<attribute key="decayTo" value="23482" />
+		<attribute key="duration" value="10" />
+		<attribute key="corpseType" value="venom" />
+		<attribute key="fluidSource" value="slime" />
+	</item>
+	<item id="23482" name="the remains of an elemental">
+		<attribute key="containerSize" value="11" />
+		<attribute key="weight" value="10000" />
+		<attribute key="decayTo" value="0" />
+		<attribute key="duration" value="500" />
+		<attribute key="corpseType" value="venom" />
+		<attribute key="fluidSource" value="slime" />
+	</item>
+	<item id="23484" article="a" name="slain devourer">
+		<attribute key="weight" value="25000" />
+		<attribute key="containerSize" value="11" />
+		<attribute key="decayTo" value="23485" />
+		<attribute key="duration" value="10" />
+		<attribute key="corpseType" value="venom" />
+		<attribute key="fluidSource" value="slime" />
+	</item>
+	<item id="23485" article="a" name="slain devourer">
+		<attribute key="containerSize" value="11" />
+		<attribute key="weight" value="10000" />
+		<attribute key="decayTo" value="0" />
+		<attribute key="duration" value="500" />
+		<attribute key="corpseType" value="venom" />
+		<attribute key="fluidSource" value="slime" />
+	</item>
+	<item id="23495" article="a" name="necrometer">
 		<attribute key="weight" value="800" />
 	</item>
-	<item id="23491" article="a" name="pint of glooth">
+	<item fromid="23496" toid="23503" article="a" name="lever" />
+	<item id="23508" article="a" name="pint of glooth">
 		<attribute key="weight" value="200" />
 	</item>
-	<item id="23497" article="a" name="glooth sandwich">
+	<item id="23514" article="a" name="glooth sandwich">
 		<attribute key="weight" value="800" />
 	</item>
-	<item id="23498" article="a" name="bowl of glooth soup">
+	<item id="23515" article="a" name="bowl of glooth soup">
 		<attribute key="weight" value="100" />
 	</item>
-	<item id="23499" article="a" name="bottle of glooth wine">
+	<item id="23516" article="a" name="bottle of glooth wine">
 		<attribute key="weight" value="400" />
 	</item>
-	<item id="23500" article="a" name="glooth steak">
+	<item id="23517" article="a" name="glooth steak">
 		<attribute key="weight" value="1200" />
 	</item>
-	<item id="23512" article="a" name="glooth spear">
+	<item id="23523" article="a" name="strangely coloured glooth">
+		<attribute key="description" value="This glooth is of a strange blue colour." />
+	</item>
+	<!--<item id="23529" article="a" name="glooth spear">
```

Comment:
@ninjalulz feel free to merge this pull request and add the glooth spear since mark added it to main tfs, I've already added everything else